inference-engine/src/mkldnn_plugin/mkldnn_memory.cpp

```diff
@@ -489,8 +489,8 @@ static const std::map<int, std::vector<mkldnn::memory::format_tag>> form_tags_by
         mkldnn::memory::format_tag::aBCde4c8b2c,
     }}, {6, {                                    // Popular
         mkldnn::memory::format_tag::abcdef,      // plain
-        mkldnn::memory::format_tag::acbdef,      // permuted
-        mkldnn::memory::format_tag::defcab,      // permuted
+        mkldnn::memory::format_tag::acbdef,      // transposed
+        mkldnn::memory::format_tag::defcab,      // transposed
```

Comment:
the same. not necessary to change the wording

Reply:
Fixed